parlai/utils/flake8.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+"""
+Custom flake8 rules for ParlAI.
+
+Includes:
+- Checks for python3 shebang
+- Check for copyright message
+- Docformatter issues (TODO)
+"""
+
+import docformatter
+import difflib
+
+PYTHON_SHEBANG = '#!/usr/bin/env python3'
+WHITELIST_PHRASES = ['Moscow Institute of Physics and Technology.']
```

Comment:
Ha - what is this whitelist phrase used for? Was there a case in which we had to make an exception for this?

Reply:
yeah there's some code owned by moscow in parlai